backend/src/bundles/resumes/content/education/education.repository.ts

```diff
@@ -0,0 +1,47 @@
+import { Guid as guid } from 'guid-typescript';
+import { type Transaction } from 'objection';
+
+import { type EducationModel } from './education.js';
+
+class EducationRepository {
+    private educationModel: typeof EducationModel;
+
+    public constructor(educationModel: typeof EducationModel) {
+        this.educationModel = educationModel;
+    }
+
+    public async create(
+        resumeId: string,
+        payload: Partial<EducationModel>,
+        transaction?: Transaction,
+    ): Promise<EducationModel> {
+        payload.resumeId = resumeId;
+        payload.id = guid.raw();
```

Comment:
The database should do this for you. Check other files as well

Reply:
As we recently decided, for now I leave it as it is. It will be fixed after performed task to add generation IDs to DB.